### PR TITLE
[vi] Probabilities and Statistics

### PR DESCRIPTION
**Pull request by tuanhm2911**
Hi,
I've finished my translation to Vietnamese with topic Probabilities. Welcome your review.
Thank you.

### CODE REVIEW

**Comment by tuanhm2911** [comment]
@shervinea can you add the labels Review wanted for me. Thank you.

**Comment by shervinea** [comment]
Thank you @tuanhm2911 for your hard work!

File: vi/cs-229-probability.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+**41. Covariance ― We define the covariance of two random variables X and Y, that we note σ2XY or more commonly Cov(X,Y), as follows:**
+
+&#10230; Hiệp phương sai - Chúng ta xác định hiệp phương sai của hai biến ngẫu nhiên X và Y, thường được kí hiệu σ2XY hay Cov(X,Y), như sau:
+
+<br>
+
+**42. Correlation ― By noting σX,σY the standard deviations of X and Y, we define the correlation between the random variables X and Y, noted ρXY, as follows:**
+
+&#10230; Mối tương quan ― Kí hiệu σX,σY là độ lệch chuẩn của X và Y, chúng ta xác định mối tương quan giữa X và Y, kí hiệu ρXY, như sau:
```
Comment (hungnguyen95) [inline vi/cs-229-probability.md]: Correlation ⇒ Hệ số tương quan

**Comment by shervinea** [comment]
Hi @tuanhm2911, would the suggestions of @hungnguyen95 sound good to you? Please feel to give your feedback when you get the time. After that, we'll be able merge all your hard work!

File: vi/cs-229-probability.md (hunk 1 above)
Comment (tuanhm2911) [reply vi/cs-229-probability.md]: Thank you for your suggestion, I will update my translation

**Comment by shervinea** [comment]
Thank you everyone for your hard work!

**Pull request merged**